processing/src/main/java/io/druid/query/filter/Filter.java

```diff
@@ -27,4 +27,12 @@
 {
   public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector);
   public ValueMatcher makeMatcher(ValueMatcherFactory factory);
+
+  public ValueMatcher makeMatcher(
```

Comment:
Added some comments to the methods in Filter and a comment on what a RowOffsetMatcher does